tests/endpoints/test_vessel_movements_vessel_status_filter.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+from tests.testcases import TestCaseUsingRealAPI
+from vortexasdk import VesselMovements
+
+
+class TestVesselMovementsRealVesselStatus(TestCaseUsingRealAPI):
+    def test_search(self):
+        v = VesselMovements().search(
+            filter_time_min=datetime(2019, 10, 1),
+            filter_time_max=datetime(2019, 10, 15),
+            filter_vessel_classes=["vlcc"],
+            filter_vessel_status="vessel_status_ballast",
+        )
+
+        assert len(v) > 50
```

Comment:
Maybe it's an overkill to add a whole test for a single filter? In any case, the reason for choosing 50 movements as a cutoff point was after the following inspection:

![vessel_status_num_check](https://user-images.githubusercontent.com/31421156/79471670-cb71b600-7ffa-11ea-9bed-140b467f7656.png)

Reply:
I agree 👍 - we could add this test method to the test_vessel_movements class, instead of adding a new test class.

Reply:
Done.